### DISCUSSION TITLE
Capitolo 4 - overflow-hidden anziché overflow-scroll - Figura 4.5

### DISCUSSION THREAD

**Discussion by fqdm** [lifeisfoo/react-guida-pratica#13] [Errori e refusi]
Per ottenere lo stile mostrato nella figura 4.5 occorre sostituire overflow-scroll con overflow-hidden (quindi senza lo scroll verticale e orizzontale)

**Comment by lifeisfoo** [comment]
Ciao,
lo scopo di quel codice non è quello di nascondere sempre le barre di scorrimento ma mostrarle solo quando c'è del contenuto che non entra nella vista.

Dato che non riesco a riprodurre il tuo problema, puoi condividere qui tutto il codice che stai usando per quell'esempio (tutti i file js e css)?

Puoi dirmi anche che browser e che sistema operativo stai usando e relative versioni?

Grazie

P.s.
C'è stata una segnalazione simile in passato, forse potrebbe esserti utile https://github.com/lifeisfoo/react-guida-pratica/discussions/10

**Comment by fqdm** [comment]
ok, non ero ancora arrivato al paragrafo "Un file CSS personalizzato" della pagina 53. Con la tua risposta l'ho notato solo adesso e in effetti viene indicato che sono presenti delle barre per lo scroll causato da overflow-scroll di Bootstrap. Mi mandava in tilt la figura 4.5 mostrata senza scroll a pagina 51.

Uso Chrome e Windows 10 e con la creazione del CSS personalizzato style.css ho risolto aggiungendo il codice indicato:

*::-webkit-scrollbar{
    display: none;
}

Grazie mille
